### PR TITLE
Perf improvements v2

### PR DESCRIPTION
**Pull request by JoviDeCroock**

### CODE REVIEW

**Review by marvinhagemeister — APPROVED** [review]
This looks great! Let's add a changeset too

File: src/util.js [hunk 1]
```diff
@@ -64,14 +65,17 @@ export function styleObjToCss(s) {
...
-				str +
-				name +
-				':' +
-				val +
-				(typeof val === 'number' && IS_NON_DIMENSIONAL.test(prop) === false
-					? 'px;'
-					: ';');
+			let suffix = ';';
+			if (SUFFIX_CACHE[name]) {
+				suffix = 'px';
```
Comment (andrewiggins) [inline src/util.js:70]: does this need to be `'px;'` (note the semicolon after the `px`)?

File: src/util.js [hunk 2]
```diff
@@ -64,14 +65,17 @@ export function styleObjToCss(s) {
...
-			str =
-				str +
-				name +
-				':' +
-				val +
-				(typeof val === 'number' && IS_NON_DIMENSIONAL.test(prop) === false
-					? 'px;'
-					: ';');
+			let suffix = ';';
+			if (SUFFIX_CACHE[name]) {
```
Comment (andrewiggins) [inline src/util.js:69]: Do we need to check if `typeof val === 'number'` here as well? If the first call to render is `<div style={{ padding: 8 }} />` but the second is `<div style={{ padding: '8px' }} />`, will it still add the `'px'`?

**Review by andrewiggins — COMMENTED** [review]
@JoviDeCroock Sorry I'm a little late to the game 😅 I did have a couple of quick questions when you get a chance

**Pull request merged**